src/popup/components/Modals/MultisigVaultCreate.vue

```diff
@@ -210,7 +210,8 @@ import { Encoded } from '@aeternity/aepp-sdk';
 
 import type { ICreateMultisigAccount, ObjectValues } from '@/types';
 import { MODAL_ADDRESS_BOOK_ACCOUNT_SELECTOR, PROTOCOLS } from '@/constants';
-import { excludeFalsy, handleUnknownError } from '@/utils';
+import { tg } from '@/popup/plugins/i18n';
```

Comment:
Why do we need to export `tg` in this file? Every other place is using `t` from `i18n` module.

Reply:
Replaced with `t`